assignment-2/assignment2.rb

```diff
@@ -0,0 +1,220 @@
+=begin
```

Comment:
first suggestion. break these up into separate files so you can run them in console. Right now they would all try to run but you have non ruby syntax where you list each assignment. consider using comment syntax `# the hash before a line comments out the text in ruby`